modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -225,8 +224,17 @@ class Create_Timepoint extends \NDB_Form
      */
     function _validate(array $val)
     {
+        $user = \User::singleton();
+
         $errors = array();
 
+        // validate site entered
+        $site = $val['psc'];
+
+        if (empty($site) || !$user->hasCenter($site)) {
+            $errors['psc'] = "Site must be selected from the available dropdown.";
+        }
+
```

Comment:
@ridz1208 
while this throws an error for users at more than one site when not setting a site at all, it will throw the same error for users at one site (empty($site) will be true) since they do not even set the site (it defaults to their one site).

Reply:
@MounaSafiHarab when user has only one site, isn't there a dropdown with only one site with that one value selected by default ?
shouldn't be returning empty lol

Reply:
@ridz1208 
no because when the user is at one site, I was asked to not change the options that get displayed on the front end for existing users, so the Site dropdown only appears if you are at more than one site

Reply:
ahhhh okai I will add a >1 check